### PR TITLE
fix(editor): editing peers required the token by accident

### PR DESCRIPTION
**Pull request by Itrulia**
Created a new function called toggleRequiredLabel to add the required symbol to labels.
This allows to have an easy way to toggle if it's required or not (adding/editing) and gives us a central way to change the icon if ever desired

### CODE REVIEW

File: packages/editor/__tests__/specs/__snapshots__/peerEditPanel.tsx.snap [hunk 1]
```diff
@@ -375,7 +375,7 @@ exports[`Peer Edit Panel should render with ID 1`] = `
               for="token"
               id="token-control-label"
             >
-              peerList.panels.token*
```
Comment (Itrulia) [inline packages/editor/__tests__/specs/__snapshots__/peerEditPanel.tsx.snap:378]: There's still a snapshot where it's present to make sure it's there

File: packages/editor/__tests__/specs/toggleRequiredLabel.test.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import {toggleRequiredLabel} from '../../src/client/toggleRequiredLabel'
```
Comment (camilleory) [inline packages/editor/__tests__/specs/toggleRequiredLabel.test.ts:1]: I'm curious about testing: is it necessary to write a test for such a small thing ? Why do you think it's necessary ?

**Review by camilleory — APPROVED** [review]
Nice solution 👍 
Should we remove the warning or the minimum length of 8 characters for passwords, as some users might have an old password which is shorter ? 
image

File: packages/editor/__tests__/specs/toggleRequiredLabel.test.ts (hunk 2 above)
Comment (Itrulia) [reply packages/editor/__tests__/specs/toggleRequiredLabel.test.ts:1]: Mistakes happen, this just makes sure they don't in the future. Especially when refactoring things like this makes it easier.

If functions are small and pure usually writing tests are quick. I use snapshots for these kind of tests to use as regression testing, as if we change the symbol then we can just press `u` to update the snapshots easily.

There's things I agree that do not need to have unit tests (getters or setters for example), but anywhere there is logic involved, tests should be present.

**Review by penk85 — APPROVED** [review]
I like the re-usability :+1:

**Pull request merged**